//羁绊信息
message LinkPB{
	required int32 link_no = 1;  // 羁绊编号
	optional bool is_activation = 2;  // 是否激活
}

//武将信息
message HeroPB
{
    required int32 hero_no = 1;
    optional int32 level = 2;
    optional int32 exp = 3;
    optional int32 break_level = 4;
    repeated LinkPB links = 5;
}